config/filter.d/drupal-auth.conf

```diff
@@ -0,0 +1,28 @@
+# Fail2Ban filter to block repeated failed login attempts to Drupal site(s)
+#
+#
+# Drupal must be setup to use Syslog, which defaults to the following format:
+#
+#   !base_url|!timestamp|!type|!ip|!request_uri|!referer|!uid|!link|!message
+#
+#
+
+[INCLUDES]
+
+before = common.conf
+
+
+[Definition]
+
+_daemon =
+
+failregex = ^%(__prefix_line)s.+\|\d{10}\|user\|<HOST>\|.+\|.+\|\d\|.?\|Login attempt failed for .+\.$
```

Comment:
I think I made it specific enough, still `.+`s after `<HOST>`, but I'm not sure what they may be (e.g. referrer, link). Also found some info that said `base_url` will never have a trailing slash, but can be a subdirectory, so I tweaked a sample log for the subdir (and tested the trailing slash exclusion myself). Ref #1035